test/e2e/e2e.go

```diff
@@ -0,0 +1,70 @@
+package test
+
+import (
+	"github.com/appscode/restik/pkg/controller"
+	"k8s.io/kubernetes/pkg/client/unversioned/clientcmd"
+	"k8s.io/kubernetes/pkg/api"
+	"time"
+	"fmt"
+	"errors"
+)
+
+func runController() (*controller.Controller, error) {
+	config, err :=  clientcmd.BuildConfigFromFlags("", "/home/sauman/.kube/config")
```

Comment:
you can't point your pc specific config. What if i try to run those test in my machine?